ipld/hamt/src/lib.rs

```diff
@@ -23,12 +23,15 @@ pub use forest_hash_utils::{BytesKey, Hash};
 use serde::{Deserialize, Serialize};
 
 pub use self::error::Error;
-pub use self::hamt::Hamt;
+pub use self::hamt::{Hamt, Hamtv0};
 pub use self::hash::*;
 pub use self::hash_algorithm::*;
+pub use self::pointer::version;
 
 /// Default bit width for indexing a hash at each depth level
 const DEFAULT_BIT_WIDTH: u32 = 8;
+/// Default bit width for indexing a hash at each depth level for Hamt v0
+pub const DEFAULT_BIT_WIDTH_V0: u32 = 5;
```

Comment:
Let's make this private unless we need it to be public for some reason.

Reply:
I've removed this field due to #1828 and https://github.com/filecoin-project/builtin-actors/issues/1346. Basically, the user should just pass the bit-width they need and I'm going to deprecate the "default" methods as they're useless.

As far as I can tell, the v0 bit-width was also 8 by default. We just set it to 5 everywhere.